services/madoc-ts/src/database/queries/deletion-queries.ts

```diff
@@ -0,0 +1,14 @@
+import { sql } from 'slonik';
+
+export function deleteIiifMetadata(id: number) {
+  return sql<{ resource_id: number; total: number }>`
+    delete from iiif_metadata where resource_id = ${id};
+  `;
+}
```

Comment:
There's 2 types of metadata. The original metadata (`resource_id = :id AND site_id is null`) and also site-specific overrides (`resource_id = :id and site_id = :sid`)

Reply:
Yeah, I noticed that when doing the metadata deletion work. Locally this function is currently looking like this:

```
export function deleteIiifMetadata(id: number, siteId?: number) {
  if (siteId) {
    return sql`
      delete from iiif_metadata where resource_id = ${id} and id = ${siteId};
    `;
  }
  return sql`
    delete from iiif_metadata where resource_id = ${id};
  `;
}
```

Reply:
Right that makes sense, so if you don't pass in a site ID you'll delete _all_ piece of metadata.